src/platform/i386/boot_comp.c

```diff
@@ -72,8 +74,8 @@ boot_pgtbl_mappings_add(struct captbl *ct, capid_t pgdcap, capid_t ptecap, const
 		paddr_t pf    = chal_va2pa(p);
 		u32_t   mapat = (u32_t)user_vaddr + i * PAGE_SIZE, flags = 0;
 
-		if (uvm && pgtbl_mapping_add(pgtbl, mapat, pf, PGTBL_USER_DEF)) assert(0);
-		if (!uvm && pgtbl_cosframe_add(pgtbl, mapat, pf, PGTBL_COSFRAME)) assert(0);
+		if (uvm && pgtbl_mapping_add(pgtbl, mapat, pf, X86_USER_DEF)) assert(0);
```

Comment:
This file is in i386, why use `X86_` prefixes? Unless this is coming from some architecture independent header. Which I don't think is the case here.

Reply:
Here I'm using the prefix to separate this from generic flags, or it will cause confusion there.